code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -108,3 +112,223 @@
 	result = list(
 	/turf/simulated/wall = 1,
 	/obj/effect/decal/cleanable/fungus = 7)
+
+
+
+// z6 DEPOT SPAWNERS
+
+/obj/effect/spawner/random_spawners/syndicate
+	spawn_nothing_percentage = 50
+
+
+// Turrets
+
+/obj/effect/spawner/random_spawners/syndicate/turret
+	name = "50pc int turret"
+	icon_state = "x"
+	result = list(
+	/obj/machinery/porta_turret/syndicate/interior  = 1
+	)
+
+/obj/effect/spawner/random_spawners/syndicate/turret/external
+	name = "50pc ext turret"
+	result = list(
+	/obj/machinery/porta_turret/syndicate/exterior  = 1
+	)
+
+
+// Mobs
+
+/obj/effect/spawner/random_spawners/syndicate/mob
+	name = "50pc melee syndimob"
+	icon_state = "x"
+	color = "#333333"
+	result = list(
+	/mob/living/simple_animal/hostile/syndicate/melee/autogib/depot = 1
+	)
+
+
+// Traps
+
+/obj/effect/spawner/random_spawners/syndicate/trap
+	icon_state = "x"
+	color = "#000000"
+
+/obj/effect/spawner/random_spawners/syndicate/trap/pizzabomb
+	name = "50pc trap pizza"
+	spawn_nothing_percentage = 0
+	result = list(
+	/obj/item/pizzabox/meat = 1,
+	/obj/item/pizza_bomb/autoarm = 1
+	)
+
+/obj/effect/spawner/random_spawners/syndicate/trap/medbot
+	name = "50pc trap medibot"
+	result = list(
+	/mob/living/simple_animal/bot/medbot/syndicate/emagged = 1
+	)
+
+/obj/effect/spawner/random_spawners/syndicate/trap/mine
+	name = "50pc trap landmine"
+	result = list(
+	/obj/effect/mine/explosive = 1
+	)
+
+/obj/effect/spawner/random_spawners/syndicate/trap/documents
+	name = "66pc trapped documents"
+	spawn_nothing_percentage = 0
+	result = list(
+	/obj/item/documents/syndicate/yellow = 1,
+	/obj/item/documents/syndicate/yellow/trapped = 2
+	)
+
+
+
+// Loot
+
+/obj/effect/spawner/random_spawners/syndicate/loot
+	name = "common loot"
+	icon_state = "x3"
+	// Loot schema: costumes, toys, useless gimmick items, trapped items
+	result = list(
+	/obj/item/storage/toolbox/syndicate = 1,
```

Comment:
I'd prefer for lists like this, that the first item is on the same line as the list, then all others are indented to it, rather than it just be an open parenthesis then a new line for the next object. Nitpicky, but I think it looks cleaner.